### PR TITLE
Include Google structured data for blogs

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-1895](https://trello.com/c/k7r8173i/1895-seo-investigate-structured-data)

### Context

Adding structured data for the blogs enables Google to enhance the search results for blog pages.

### Changes proposed in this pull request

- Include Google structured data for blogs

### Guidance to review

Tested using the Google structured data tool (the warning doesn't apply as we don't have author pages):

Screenshot 2021-09-02 at 10 19 01

Example of how the blog post will display in Google search results:

non-amp-article

### CODE REVIEW

File: app/helpers/structured_data_helper.rb [hunk 1]
```diff
@@ -18,6 +18,25 @@ def structured_data(type, data)
...
+    frontmatter = page.frontmatter
+
+    data = {
+      headline: frontmatter[:title],
+      image: frontmatter[:images].values.map { |h| h["path"] },
+      datePublished: frontmatter[:date],
+      keywords: frontmatter[:keywords],
+      author: [
+        {
+          "@type": "Organization",
```
Comment (jkempster34) [inline app/helpers/structured_data_helper.rb:31]: Not sure if it matters, but could the author type be conditional (Organisation or Person) based on whether the blog author has defaulted to "Get into Teaching"? https://github.com/DFE-Digital/get-into-teaching-app/blob/dc56f963d24db8d41acbc5e7a8d989c08cb32d2c/app/views/blog/_date_and_author.html.erb#L6

**Review by jkempster34 — COMMENTED** [review]
Not sure if it matters, but could the author type be conditional based on whether the

File: app/helpers/structured_data_helper.rb (hunk 1 above)
Comment (ethax-ross) [reply app/helpers/structured_data_helper.rb:31]: Yeah that's a good shout, I had thought all the blog posts were `author: Get Into Teaching` but some are missing the key entirely and others are actual names. I'll clarify it with the content team and update the PR.

**Review by jkempster34 — APPROVED** [review]
LGTM! 💯

**Pull request merged**